doxygen/qdx

```diff
@@ -230,6 +230,58 @@ class qdx {
         return comment;
     }
 
+    #! doxygen dislikes qouted strings but accept apostrophed ones
+    string adjustQuotedStrings(string line) {
```

Comment:
Will this  change strings in every documentation comments? I'm really against such pre/post processing of documentation, it may change the meaning of some comments, examples etc where quoted strings were intentionally used. Do you have some examples where it generates warnings or so? shouldn't we fix the documentation there instead of this automatic changes?

Reply:
sure not, it is related to argument default vaues and it is obvious and commented when called